### PR TITLE
[fix #246] fix unified sorter

### LINKED ISSUE #246: cdc: unified sorting is not working properly
### 1. Describe the bug
From metric, the preparation time for the sorting of `unified sort` is too long, 30G data took 7 minutes.

### 2. Minimal reproduce step (Required)
1. get a tso for now
2. put 30G data into upstream tikv
3. create a changefeed with tso got from step1
4. watch metric

### 3. What did you see instead (Required)
Sorter takes 7 minutes before output data
 
### 4. What did you expect to see? (Required)
Sorter takes less than 1 minutes before output data

### 5. What is your migration tool and TiKV version? (Required)

- TiKV CDC: master

### PR DESCRIPTION
**Pull request by zeminzhou**
Signed-off-by: zeminzhou <zhouzemin@pingcap.com>

### What problem does this PR solve?
1. fix #issue246
2. remove unused log
3. fix metric for `puller output resolvedts`

Issue Number: close #246

Problem Description: **TBD**

### What is changed and how does it work?
remove `wrong print`

### Code changes

- Has exported function/method change

### Check List for Tests

This PR has been tested by at least one of the following methods:
- Unit test
- Integration test
- Manual test

### CODE REVIEW

**Comment by zeminzhou** [comment]
@PTAL~

**Comment by zeminzhou** [comment]
PTAL~

File: cdc/cdc/puller/puller.go [hunk 1]
```diff
@@ -231,6 +227,7 @@ func (p *pullerImpl) Run(ctx context.Context) error {
 				if err != nil {
 					return errors.Trace(err)
 				}
+				metricTxnCollectCounterResolved.Inc()
```
Comment (pingyu) [inline cdc/cdc/puller/puller.go:230]: Suggest:
1. Rename to `metricPullOutputCounterResolved`. So as to the underlying prom metric name.
2. Utilize [`kvEventCounter`](https://github.com/tikv/migration/blob/b559233b0f166429b904b8a4390db310a966dacb/cdc/cdc/puller/metrics.go#L21) as the puller input counter. The input & output metrics will help us determine whether there is issue in puller module. For both kv & resolved-ts event.
3. Add the input counter to grafana panel.

File: cdc/metrics/grafana/tikv-cdc.json [hunk 2]
```diff
@@ -6496,100 +6496,6 @@
             "alignLevel": null
           }
         },
-        {
```
Comment (haojinming) [inline cdc/metrics/grafana/tikv-cdc.json:6499]: Removing a panel will leave a blank space, may need move all other panels or modify the metric json in next pr?

Comment (zeminzhou) [reply cdc/metrics/grafana/tikv-cdc.json]: yesh, it will leave a blank space. I add it back, and remove it in next pr. Thanks

**Review by pingyu — COMMENTED** [review]
Please paste the screenshot of modified `TiKV-CDC` panel of Grafana in PR content.
Rest LGTM~

**Review by haojinming — APPROVED** [review]
LGTM

**Comment by zeminzhou** [comment]
image
image
image

**Comment by zeminzhou** [comment]
> Please paste the screenshot of modified `TiKV-CDC` panel of Grafana in PR content. Rest LGTM~

added, thanks

**Review by pingyu — APPROVED** [review]
LGTM~

**Pull request merged**